scripts/remove_after_use/revoke_figshare.py

```diff
@@ -28,14 +28,14 @@ def main(guids, dry=False):
     with transaction.atomic():
```

Comment:
If this script has been used, it can probably just be removed instead of updated

Reply:
Good point; I'll clear out this directory now.